2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -822,4 +822,132 @@ La timeline permette di avere una visione delle attività incentrata sulle tempi
 La timeline permette di mostrare il grafico di Gantt delle attività evidenziando i rapporti di dipendenza tra le task e stabilendo le scadenze necessarie per il loro svolgimento. 
 ===== Grafici
 Jira offre la possibilità di produrre grafici e report relativi all'avanzamento e alla tracciabilità delle task. Tali strumenti permettono di avere delle metriche di valutazione dell'andamento del progetto e di individuare eventuali criticità.
-Il gruppo utilizza come metrica principale il burndown chart, che permette di avere una visione dell'avanzamento delle attività in base al tempo, basato sugli story points di ogni attività.
\ No newline at end of file
+Il gruppo utilizza come metrica principale il burndown chart, che permette di avere una visione dell'avanzamento delle attività in base al tempo, basato sugli story points di ogni attività.
+
+== Processo di gestione delle informazioni
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.6_
+=== Scopo
+Lo scopo della gestione delle informazionie è generare, ottenere, confermare, trasformare, conservare, recuperare, distribuire le informazioni e la relativa documentazione agli stakeholder interessati, garantendone chiarezza, completeszza, consistenza, tracciabilitò e presentazione.
+
+Le informazioni possono essere tecniche, di progetto, organizzative e di accordo.
+
+=== Informazioni gestite 
+Le informazioni gestite dal gruppo sono:
+- documentazione:
+  - Piano di Progetto;
+  - Norme di Progetto;
+  - Piano di Qualifica;
+  - Analisi dei Requisiti;
+  - Glossario;
+  - Verbali.
+- codice sorgente:
+  - Proof of Concept;
+  - Product Baseline.
+
+Tali documenti sono creati, organizzati, aggiornatti, versionati e distribuiti all'interno dei repository pubblico del gruppo. 
+
+=== Documentazione
+==== Struttura <struttura-documenti>
+Ogni documento segue una struttura standard, stabilita nel template _template.typ_. I documenti pertanto sono così strutturati:
++ *Cover page*: la cover page è la prima pagina del documento, e contiene le seguenti informazioni:
+  - nome del gruppo;
+  - logo;
+  - titolo del documento;
+  - sottotitolo del documento;
+  - versione;
+  - stato;
+  - tipo di uso: interno od esterno;
+  - responsabile del gruppo;
+  - redattori;
+  - verificatori;
+  - destinatari;
+  - figure esterne se presenti.
+
++ *Indici*: sezione successiva alla cover page. Sono presenti tre tipologie di indici:
+    - indice del contenuto: indice sempre presente che rappresenta i paragrafi del documento;
+    - indice delle tabello: indice presente solo se sono presenti tabelle nel documento;
+    - indice delle figure: indice presente solo se sono presenti figure nel documento.
+
++ *Tracciamento modifiche*: sezione successiva agli indici. Maggiori dettagli sono visibili alla sezione dedicata (@tracciamento-modifiche).
```

Comment:
Nei file si chiama "Registro delle modifiche". Lo sposterei prima degli "Indici" così da andare in ordine di comparsa nel documento